test/unittests/mongoBackend/mongoUpdateContext_test.cpp

```diff
@@ -9199,110 +9199,6 @@ TEST(mongoUpdateContextRequest, servicePathEntityUpdate_3levels)
 
 /* ****************************************************************************
 *
-* servicePathEntityUpdates_2levels -
-*
-*/
-TEST(mongoUpdateContextRequest, servicePathEntityUpdate_2levels)
```

Comment:
No, I think you have understood it perfectly. I was missing al the "context" (prepareDatabase, etc.), but after reading your explanation makes fully sense to remove the tests.

NTC